### DISCUSSION TITLE
Getting undefined method error when calling getBudget() and getBudgets()

### DISCUSSION THREAD

**Discussion by adjuto** [XeroAPI/xero-php-oauth2#313] [Q&A]
Am using PHP SDK and API quite successfully in most areas but getting fatal error when attempting to call both getBudgets() and getBudget() in Accounting.

Fatal error: Uncaught Error: Call to undefined method XeroAPI\XeroPHP\Api\AccountingApi::getBudget()  and
XeroAPI\XeroPHP\Api\AccountingApi::getBudgets()

Reverted to Xero examples and get same error. API Explorer working find and showing data in my Organisation.

Contacted Xero support and they recommended raising question here. Suggested that complexities of dlls could be linked to this?

Has anyone else experienced the same? Any thoughts / ideas much appreciated. Not sure where to go at this point. Many thanks

**Comment by RettBehrens** [comment]
Hi @adjuto what version of the SDK are you running? From the description it sounds like you might be running a version that predates the addition of these methods 🤔

**Reply by RettBehrens** [reply]
Ah, yep confirmed. Found your case with API-CX and checked your app logs. Per the logs you're using v2.5.4 - the methods you're trying to use were added with the v2.6.0 release

**Reply by adjuto** [reply]
Many thanks for this. I will make the changes to use v2.6.0

**Reply by adjuto** [reply]
It worked! All sorted - thanks again for your help
